### DISCUSSION TITLE
Add the ability to create references inside footnotes and captions

### DISCUSSION THREAD

**Discussion by jeffpooley** [knowledgefutures/pubpub#2234] [Ideas]
I am hoping to re-surface this citation/footnote issue that @gabestein and I discussed in this this DOI Improvements [thread](https://github.com/pubpub/pubpub/discussions/2126). (It is also related to to this [Enhancement Request](https://github.com/pubpub/pubpub/issues/1281) to support citations in captions.)

Here’s the issue: Right now PubPub’s built-in citation feature can’t be used in footnotes nor in captions. I had thought that my interest in Chicago note+bibliography needs were a humanities edge case. But working on some Pubs using APA style recently, I realized that the problem is broader.

Here’s an [example Pub](https://www.mediastudies.press/pub/gross-preface-2022/draft?access=3wvyqeyd). The author has included in-text APA references in footnotes, in addition to placing them in the main text. The problem is that an in-text reference like “Maslow (1954)” can’t be made to anchor a citation when it’s in a footnote. The very hacky workaround I’ve used is to create invisible citations after the footnote marker, by adding a single space in the citation window’s “Inline Format” (custom) field. This breaks Word exports, however, and creates a confusing/overlapping series of footnote and citation popovers. 

<img width="487" alt="citation-exmaple" src="https://user-images.githubusercontent.com/6372329/188887705-b72adc06-a1c7-4caa-8410-960b9cfc6e9c.png">

The solution, which may be a big lift, would be to add the ability to create citations *inside* footnotes and image captions. So that when “Maslow (1954)” appears in a footnote, I can use that as an anchor for the reference in the same way I can do that in the main text.

**Comment by tosteiner** [comment]
Thanks, Jeff - totally agree, this would very much improve how citations can be used! 

Just to add other potential leads that might help with implementation, we had previously flagged this issue in an early write-up of using [PubPub for scholarly writing](https://copim.pubpub.org/pub/using-pubpub-for-scholarly-writing-citations-collaboration-zotero#njy7kvgndai), and next to that, there seems to be an additional issue that Gabe had flagged, see #1861 - both of which seem to nicely segue over to a potential further development line that would hopefully bring PubPub's citation engine closer to a cross-integration with Zotero (also see #1568 ) 😇

[EDIT] Also just stumbled over this other idea that seems to raise the same issue - #1760
